winter-framework/src/main/java/de/uni_mannheim/informatik/dws/winter/matching/rules/LinearCombinationMatchingRule.java

```diff
@@ -120,73 +121,73 @@ public void normalizeWeights() {
 	}
 
 	@Override
-	public Correspondence<RecordType, SchemaElementType> apply(RecordType record1, RecordType record2,
-			Processable<Correspondence<SchemaElementType, Matchable>> schemaCorrespondences) {
+	public Correspondence<RecordType, SchemaElementType> apply(RecordType record1, RecordType record2, Processable<Correspondence<SchemaElementType, Matchable>> schemaCorrespondences) {
 
-		// double similarity = compare(record1, record2, null);
-		LinkedList<String> detailedResults = new LinkedList<String>();
+//		double similarity = compare(record1, record2, null);
 		double sum = 0.0;
+		Record model = null;
+		if(this.isCollectDebugResults()){
+			model = new Record(record1.getIdentifier() + "-" + record2.getIdentifier());
+			model.setValue(this.getComparatorLog().getSchema().getRecord("MatchingRule"), getClass().getName());
+			model.setValue(this.getComparatorLog().getSchema().getRecord("Record1Identifier"), record1.getIdentifier());
+			model.setValue(this.getComparatorLog().getSchema().getRecord("Record2Identifier"), record2.getIdentifier());
+		}
 		for (int i = 0; i < comparators.size(); i++) {
 			Pair<Comparator<RecordType, SchemaElementType>, Double> pair = comparators.get(i);
 
 			Comparator<RecordType, SchemaElementType> comp = pair.getFirst();
-
-			Correspondence<SchemaElementType, Matchable> correspondence = getCorrespondenceForComparator(
-					schemaCorrespondences, record1, record2, comp);
-
+ 
+			Correspondence<SchemaElementType, Matchable> correspondence = getCorrespondenceForComparator(schemaCorrespondences, record1, record2, comp);
+			
 			double similarity = comp.compare(record1, record2, correspondence);

```

Comment:
log comparator results to separate log in short format (one line per comparator)

Reply:
Done